langchain4j-milvus/src/main/java/dev/langchain4j/store/embedding/milvus/MilvusEmbeddingStoreExtend.java

```diff
@@ -0,0 +1,46 @@
+package dev.langchain4j.store.embedding.milvus;
+
+import dev.langchain4j.data.embedding.Embedding;
+import dev.langchain4j.store.embedding.EmbeddingMatch;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+
+import java.util.List;
+
+/**
+ * Milvus Embedding Store extend
+ */
+public interface MilvusEmbeddingStoreExtend<Embedded> extends EmbeddingStore<Embedded> {
```

Comment:
Is this interface really required? Can't you just define these methods in `MilvusEmbeddingStore`?